packages/strapi-design-system/src/Alert/Alert.stories.mdx

```diff
@@ -13,95 +14,97 @@ import { Link } from '../Link';
+<Canvas withSource="open">
+  <Story
+    name="default"
+    parameters={{
+      docs: {
+        source: {
+          code: '<Alert title="Title">This is the default alert.</Alert>',
+        },
+      },
+    }}
+  >
+    <Box style={{ width: 700 }}>
+      <Alert title="Title">This is the default alert.</Alert>
+    </Box>
+  </Story>
+</Canvas>
+
+### Alert variants
+
+Use the variant prop to change the visual style of the Button.
+You can set the value to `default`, `success`, or `danger`.
+
+<Canvas withSource="open">
+  <Story
+    name="variants"
+    parameters={{
+      docs: {
+        source: {
+          code: '<Alert title="Title" variant="success">This is the default variant.</Alert>',
+        },
+      },
+    }}
+  >
+    <Box style={{ width: 700 }}>
+      <Stack size={1}>
+        <Alert title="Title">This is the default variant.</Alert>
+        <Alert title="Title" variant="success">
+          This is the success variant.
         </Alert>
-        <Alert
-          title="Danger variant"
-          variant="danger"
-          onClose={() => {}}
-          closeLabel="Close notification"
-          action={<Link to="/somewhere">See more</Link>}
-        >
-          Alert with title and longer description
+        <Alert title="Title" variant="danger">
+          This is the danger variant.
         </Alert>
       </Stack>
     </Box>
   </Story>
 </Canvas>
 
-## Alert
+### Alert with action
 
 Description...
 
-<Canvas>
-  <Story name="live region">
-    {() => {
-      const [visible, setVisible] = useState(false);
-      return (
-        <Box padding={10} style={{ width: 700 }}>
-          <Stack size={4}>
-            <button onClick={() => setVisible((s) => !s)}>Open alert</button>
-            {visible && (
-              <Alert title="Default variant" onClose={() => {}} closeLabel="Close notification" variant="danger">
-                Alert with title and longer description, lorem ipsum dolor sit amet constrectum adipisicng lorem ipsum
-                dolor sit amet consrectumis adipisingus.
-              </Alert>
-            )}
-          </Stack>
-        </Box>
-      );
+<Canvas withSource="open">
+  <Story
+    name="with-action"
+    parameters={{
+      docs: {
+        source: {
+          code: `
+<Alert title="This is the title of the alert" action={<Link to="/somewhere">See more</Link>}>
+  Alert with title and longer description, lorem ipsum dolor sit amet constrectum adipisicng lorem ipsum dolor sit
+  amet consrectumis adipisingus.
+</Alert>`,
+        },
+      },
     }}
+  >
+    <Box style={{ width: 700 }}>
+      <Alert title="This is the title of the alert" action={<Link to="/somewhere">See more</Link>}>
+        Alert with title and longer description, lorem ipsum dolor sit amet constrectum adipisicng lorem ipsum dolor sit
+        amet consrectumis adipisingus.
+      </Alert>
+    </Box>
   </Story>
 </Canvas>
+
+## Accessibility
+
+- ...
```

Comment:
@mfrachet I will need your help here 😅 
I thought this component was provided with some live-region props 🤔  and an alert/alertdialog role.

Reply:
TBH, I wasn't sure about how it would be used, and did not put a role on it. Depending on the usage we have, it will eventually require adjustments. That's a great question 😊

Reply:
It has a role :) . Check line 75 of the Alert component (on AlertBody)